### DISCUSSION TITLE
/bill命令这几天是不是不能得到结果了？

### DISCUSSION THREAD

**Discussion by Mjmaster** [TBXark/ChatGPT-Telegram-Workers#207] [General]
这几天输入/bill后，“已使用”和“剩余额度”都显示$NaN

📊 本月机器人用量

 - 总额度: $25.00
 - 已使用: $NaN
 - 剩余额度: $NaN

**Comment by Silentely** [comment]
对，现在是直接显示0

**Comment by tbxark** [comment]
Open AI 的 API 发生变动，现在不能用APIKey访问里。我看看能怎么改一下不，

**Comment by tbxark** [comment]
解决不了，这个就是被限制了，得要session key才能访问。session key获取实在是太麻烦了，我直接干掉这个功能了。
